src/index.html

```diff
@@ -11,7 +11,182 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
```

Comment:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
      <fieldset>
```
no need to add a blank line between parent and child elements, only between siblings